src/lib/Preview.js

```diff
@@ -538,14 +543,14 @@ class Preview extends EventEmitter {
                 params,
             );
 
-            DownloadReachability.downloadWithReachabilityCheck(downloadUrl);
+            this.api.reachability.downloadWithReachabilityCheck(downloadUrl);
 
             // Otherwise, get the content download URL of the original file and download
         } else {
             const getDownloadUrl = appendQueryParams(getDownloadURL(this.file.id, apiHost), queryParams);
-            api.get(getDownloadUrl, { headers: this.getRequestHeaders() }).then(data => {
+            this.api.get(getDownloadUrl, { headers: this.getRequestHeaders() }).then(data => {
```

Comment:
In a follow-up, we could probably generalize this to `this.api.file.getDownloadUrl` and hide the reachability checks from the consumer entirely.